args/core/time/time_span.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <chrono>
+#include <core/types/primitives.hpp>
+
+namespace args::core::time
+{
+	template<typename time_type>
+	class time_span
+	{
+	private:
+		std::chrono::duration<time_type> m_duration;
+
+	public:
+		time_span(std::chrono::duration<time_type> duration) : m_duration(duration) {}
+
```

Comment:
as per cppreference.org I require 
`microfortnights`(std::ratio<12096,10000>) and
`nanocenturies` (std::ratio<3155,1000>)